apps/ledger-live-mobile/e2e/page/manager/manager.page.ts

```diff
@@ -27,7 +33,11 @@ export default class ManagerPage {
   }
 
   async waitForManagerPageToLoad() {
-    await waitFor(this.managerTitle()).toBeVisible();
+    await waitForElementById(this.managerTitleId);
+  }
+
+  async expectManagerPage() {
+    await expect(getElementById(this.managerTitleId)).toBeVisible();
   }
 
   async selectSetupNewDevice() {
```

Comment:
same question

Reply:
It's part of a path that should be tested